### PR TITLE
IGVF-476-reverse-link-file-sets

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

File: src/igvfd/tests/test_types_sample.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import pytest
+
+
+def test_summary(testapp, tissue, measurement_set, analysis_set_base, curated_set_genome):
```
Comment (gabdank) [inline src/igvfd/tests/test_types_sample.py]: why is the test called "summary"?

File: src/igvfd/tests/test_types_sample.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        }
+    )
+    testapp.patch_json(
+        curated_set_genome['@id'],
+        {
+            'samples': [tissue['@id']]
+        }
+    )
+    res = testapp.get(tissue['@id'])
+    assert res.json.get('file_sets') == [measurement_set['@id'], analysis_set_base['@id'], curated_set_genome['@id']]
```
Comment (gabdank) [inline src/igvfd/tests/test_types_sample.py:26]: I am not sure you can assume the order in the list will be the same

**Review by gabdank — CHANGES_REQUESTED** [review]

File: src/igvfd/tests/test_types_sample.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        }
+    )
+    testapp.patch_json(
+        curated_set_genome['@id'],
+        {
+            'samples': [tissue['@id']]
+        }
+    )
+    res = testapp.get(tissue['@id'])
+    assert set(res.json.get('file_sets')) == set(
```
Comment (gabdank) [inline src/igvfd/tests/test_types_sample.py:26]: the right side of the equation should be `set(measurment_set['id'],,,,....`. there is no need for [] inside the set()

**Review by gabdank — CHANGES_REQUESTED** [review]

File: src/igvfd/tests/test_types_sample.py (hunk 3 above)
Comment (ian-whaling) [reply src/igvfd/tests/test_types_sample.py:26]: this change throws and error:
>       assert set(res.json.get('file_sets')) == set(measurement_set['@id'], analysis_set_base['@id'], curated_set_genome['@id'])
E       TypeError: set expected at most 1 argument, got 3

set() is used for type casting, I don't think it defines a set, { } can define a set.

**Review by gabdank — APPROVED** [review]

**Pull request merged**